ultraplot/axes/cartesian.py

```diff
@@ -563,6 +563,17 @@ def _is_panel_group_member(self, other):
             and other._panel_parent is self._panel_parent  # other is sibling panel
         )
 
+    def _safe_share(self, axis_name, other_axis):
+        if hasattr(self, f"_share{axis_name}"):
+            setattr(self, f"_share{axis_name}", None)
+        getattr(self, f"share{axis_name}")(other_axis)
```

Comment:
Where is this function used? I cannot find it used in the diff, but I may have missed it. If it is not used, we should remove it. Otherwise, we need to use it somewhere, but IDK where it should go.

Reply:
Ah this should be removed! Apologies